apps/server/src/shared/domain/service/task-copy.service.spec.ts

```diff
@@ -340,128 +332,5 @@ describe('task copy service', () => {
 				expect(task.lesson).toBeUndefined();
 			});
 		});
-
```

Comment:
Why did you remove these?

Reply:
You are right: these can be removed, as only file-copy-append-services takes care for the attached files... and these different cases are tested there.